incipyt/actions/abc.py

```diff
@@ -0,0 +1,30 @@
+class Action:
```

Comment:
While I like the idea of a base class for actions, I'm unsure about this `abc` module.

- Because `Action` is *not* an abstract class right now, as child classes do not need to implement any method
- Because the module is a bit "lost" in the middle of modules implementing concrete child classes.

Maybe we can move this to `incipyt/_internal/actions.py`?

Reply:
Fair enough. What about `_base` then?

Reply:
Rename `abc.py` to `base.py` ?

Reply:
`base.py` or `_base.py`, at your convenience :)